### DISCUSSION TITLE
Error con Azure

### DISCUSSION THREAD

**Discussion by gersondlarosa** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#572] [Q&A]
Buena tarde compañero al seguir el tutorial de implementación me arrojo el siguiente error, alguien tendrá alguna idea de que esta pasando

![2](https://user-images.githubusercontent.com/68626555/161331786-5a5dc30c-b5f6-44a0-a419-0013c71bc8de.png)
r

**Comment by doguedogue** [comment]
Estas usando las páginas estáticas? no me ha pasado en los dos proyectos que he subido algo asi.

Has usado el netlify igual es super fácil solo subir la carpeta dist creada con (`npm run build`)

**Reply by gersondlarosa** [reply]
no netlify no lo he ocupado este comando lo correo en mi proyecto npm run build y no e subido pagina estática croe que lo are
